plugin/upgrade.xml

```diff
@@ -155,4 +155,22 @@
     <migrate-command name="Run PMREP Command"/>
     <migrate-command name="Validate Deployment Group"/>
   </migrate>
+   <migrate to-version="20">
```

Comment:
Please add the following code segment in between lines 157 and 158. 
`<migrate to-version="18">`
`</migrate>`
`<migrate to-version="19">`
`</migrate>`